tests/tests_main.cpp

```diff
@@ -0,0 +1,4 @@
+#include "tests_main.h"
+#include <QTest>
+
+QTEST_APPLESS_MAIN( QskRgbLiterals )
```

Comment:
g++ --version
g++ (Debian 10.2.1-6) 10.2.1 20210110
Copyright (C) 2020 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

g++ -c -pipe -isystem /opt/qskinny/config/qsk3/Qt/include/QtCore -isystem /opt/qskinny/config/qsk3/Qt/include/QtCore/6.4.2/QtCore -isystem /opt/qskinny/config/qsk3/Qt/include/QtGui -isystem /opt/qskinny/config/qsk3/Qt/include/QtGui/6.4.2/QtGui -isystem /opt/qskinny/config/qsk3/Qt/include/QtQuick -isystem /opt/qskinny/config/qsk3/Qt/include/QtQuick/6.4.2/QtQuick -isystem /opt/qskinny/config/qsk3/Qt/include/QtQml -isystem /opt/qskinny/config/qsk3/Qt/include/QtQml/6.4.2/QtQml -pedantic-errors -Wextra -Werror=format-security -Wsuggest-override -Wno-deprecated-copy -Wno-deprecated-declarations -g -O0 -std=c++1z -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fPIC -D_REENTRANT -DQT_STRICT_ITERATORS -DITEM_STATISTICS=1 -DQT_NO_KEYWORDS -DQT_QUICK_LIB -DQT_OPENGL_LIB -DQT_GUI_LIB -DQT_TESTLIB_LIB -DQT_QMLMODELS_LIB -DQT_QML_LIB -DQT_QMLINTEGRATION_LIB -DQT_NETWORK_LIB -DQT_CORE_LIB -DQT_TESTCASE_BUILDDIR='"/opt/qskinny/build/qsk3/tests"' -I/opt/qskinny/qsk3/tests -I. -I/opt/qskinny/qsk3/src/common -I/opt/qskinny/qsk3/src/controls -I/opt/qskinny/qsk3/src/dialogs -I/opt/qskinny/qsk3/src/graphic -I/opt/qskinny/qsk3/src/inputpanel -I/opt/qskinny/qsk3/src/layouts -I/opt/qskinny/qsk3/src/nodes -I/opt/qskinny/config/qsk3/Qt/include -I/opt/qskinny/config/qsk3/Qt/include/QtQuick -I/opt/qskinny/config/qsk3/Qt/include/QtOpenGL -I/opt/qskinny/config/qsk3/Qt/include/QtGui -I/opt/qskinny/config/qsk3/Qt/include/QtTest -I/opt/qskinny/config/qsk3/Qt/include/QtQmlModels -I/opt/qskinny/config/qsk3/Qt/include/QtQml -I/opt/qskinny/config/qsk3/Qt/include/QtQmlIntegration -I/opt/qskinny/config/qsk3/Qt/include/QtNetwork -I/opt/qskinny/config/qsk3/Qt/include/QtCore -Imoc -I/opt/qskinny/config/qsk3/Qt/mkspecs/linux-g++ -o obj/tests_main.o /opt/qskinny/qsk3/tests/tests_main.cpp
/opt/qskinny/qsk3/tests/tests_main.cpp:4:36: error: ISO C++11 requires at least one argument for the "..." in a variadic macro
    4 | QTEST_APPLESS_MAIN( QskRgbLiterals )
      |                                    ^
make: *** [Makefile:466: obj/tests_main.o] Error 1